### PR TITLE
Add Github issue templates

### PR DESCRIPTION
**Pull request by junjizhi**
Add issue templates that help us structure the discussions, cut through the noise and push the project forward.

I added two templates. Feel free to add more if needed.

### CODE REVIEW

**Comment by snario** [comment]
_We need to figure out how to get CircleCI to run builds on non-organization-member pull requests_

**Comment by junjizhi** [comment]
> _We need to figure out how to get CircleCI to run builds on non-organization-member pull requests_

Yeah. From my experience CircleCI doesn't support non-org members building. 🤷‍♂

**Comment by kerzhner** [comment]
`statechannels` circleci setting was to *not* build forked pull requests (which is the default setting). I flipped the setting. After closing and re-opening this PR, the CI workflow was triggered. I am hoping that for future PR's from non-org members, CI workflow will trigger automatically. Looking forward to (and appreciate) your contributions @junjizhi.

**Comment by junjizhi** [comment]
https://circleci.com/gh/statechannels/monorepo/10887#config/containers/0

^ It seems an environment variable `NODE_ENV` was missing in the config. Not sure how your CI passed in your other merged PRs (I couldn't see those CI build details). Could you do the open-reopen trick again to trigger the build? @kerzhner

**Comment by kerzhner** [comment]
@junjizhi would you be able to rebase your PR using the latest `master` branch?

**Comment by junjizhi** [comment]
@kerzhner I did. This PR has already been rebased on latest master.

**Comment by junjizhi** [comment]
@kerzhner You sent an invite to the repo, but when I clicked to accept the link and nothing happened. 🤣

File: .circleci/config.yml [hunk 1]
```diff
@@ -178,6 +178,7 @@ jobs:
       USE_GANACHE_DEPLOYMENT_CACHE: true
       GANACHE_CACHE_FOLDER: ../../.ganache-deployments
       GANACHE_PORT: 8547
```
Comment (kerzhner) [inline .circleci/config.yml]: @junjizhi `NODE_ENV` is set to `test` in the CircleCI project settings. So, in theory, we don't need to set it here.

Comment (junjizhi) [reply .circleci/config.yml]: Yeah. Seems like I couldn't see or change any CircleCI settings for my fork repo builds. I guess all of your development is based on this monorepo, and nobody in your team used the fork-based workflow before?

**Comment by kerzhner** [comment]
FYI, you _should_ be able to see (and maybe even rerun) CircleCI jobs and workflows without needing to close + reopen the PR: https://circleci.com/gh/statechannels/workflows/monorepo. Let me know if you are not able to open that link.

**Comment by junjizhi** [comment]
@kerzhner I could see the CI but couldn't rerun. The Rerun drop-down items were all greyed out. I think it had something to do with me not being a member of the org.

File: .github/ISSUE_TEMPLATE/need-help.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+If you have a question about State Chanles that is not a bug report or feature
```
Comment (lalexgap) [inline .github/ISSUE_TEMPLATE/need-help.md]:
```suggestion
If you have a question about State Channels that is not a bug report or feature
```

**Comment by kerzhner** [comment]
I understand why CircleCI is failing on your PR: https://circleci.com/docs/2.0/oss/#pass-secrets-to-builds-from-forked-pull-requests. Environment variables are not passed to the build.

**Comment by junjizhi** [comment]
@kerzhner Are you a CircleCI admin? Can you check if the settings for forked PRs are right? https://circleci.com/docs/2.0/oss/#pass-secrets-to-builds-from-forked-pull-requests

**Pull request merged**